types/node/common.d.ts

```diff
@@ -0,0 +1,8933 @@
+// NOTE: These definitions support NodeJS 10 and all TypeScript versions.
```

Comment:
I would not write NodeJS 10 here as this will be NodeJs 11 soon and most likely forgotten to be adapted.